Changelog.md

```diff
@@ -19,6 +19,7 @@ Bugfixes:
  * Type system: Disallow ``var`` declaration with empty tuple type.
  * Type system: Correctly convert function argument types to pointers for member functions.
  * Type system: Move privateness of constructor into AST itself.
+ * Type system: Support scientific notation in numbers (e.g. ``2e8`` and ``200e-2``).
```

Comment:
It is a bugfix in the sense the parser supported it. Not sure if it was ever supported by the type system - assuming it was, this is a bug fix, otherwise it is a feature.